.github/workflow-scripts/gh-pages/deploy.js

```diff
@@ -75,7 +75,17 @@ async function deploy({ branch, sha, repo, owner, docsPath, github, exec }) {
     if (fs.existsSync(docsPath)) {
       // this is to avoid any merge conflict when an environment is redeployed
       console.info(`ℹ️ Cleaning existing ${docsPath}`);
-      await exec.exec('rm', ['-rf', docsPath]);
+      if (branch === 'master') {
+        // for master we want to clean everything from `./docs` with the exception of the directories of the deployed branches
+        const filesToRemove = fs.readdirSync('./docs', { withFileTypes: true }).filter(item => !item.name.startsWith('sb-')).map(({ name }) => name);
+        for (const file of filesToRemove) {
```

Comment:
We usually put newlines before code blocks

Reply:
Updated